ophyd_basler/tests/test_emulated_camera.py

```diff
@@ -14,6 +14,9 @@ def test_emulated_basler_camera(RE, db, make_dirs):
     print(ophyd_basler.available_devices())
 
     emulated_basler_camera = BaslerCamera(cam_num=0, verbose=True, name="basler_cam")
+
+    emulated_basler_camera.exposure_time.put(0.1)  # set exposure time to 0.1 seconds
```

Comment:
I think we can have 2 individual tests with RunEngine - one with a short acquisition that you already have, and another one with a long one (2s or so). Could you please parameterize the test to use these 2 settings? Something like we do in https://github.com/NSLS-II/sirepo-bluesky/blob/da7a04f938341bb7abe90e230b718787677a1db6/sirepo_bluesky/tests/test_bl_elements_as_ophyd_objs.py#L33.